teps/0053-nested-triggers.md

```diff
@@ -201,4 +202,86 @@ spec:
 Notice how simple it is for the Trigger author to write the logic around their trigger
 when the common logic is implemented at the top level. Trigger authors can also manage
 their own set of downstream triggers as you can specify additional triggers alongside
-a binding and template once the interceptor chain completes.
\ No newline at end of file
+a binding and template once the interceptor chain completes.
+
+## Alternatives
+
+There are three alternatives that have been identified to provide some of the same benefits.
+
+### Option 1: Select Downstream Triggers With Labels
+
+Add a `triggerSelector` field that allows a trigger to target downstream triggers based off a label selector:
+
+```yaml
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: top-level-trigger
+spec:
+  interceptors:
+    - cel:
+        filter: >-
+          header.match('header-filter', 'a-header-filter')
+        - key: example-overlay
+          expression: body.example.value
+  triggerSelector:
```

Comment:
I think this is an improvement over the original since this way the top level Trigger does not have to be updated each time a new nested Trigger is added.

Reply:
This is not necessarily orthogonal to the main proposal, it could be that we support two ways of specifying triggers, by selector and with a list. That would be consistent with the options available for selecting triggers in the event listener.